### PR TITLE
Fix various lint issues

### PR DESCRIPTION
**Pull request by chambridge**
This resolves many of lint issues in the current code base.

I've opened the following issues to evaluate the `pylint: disable` comments that were added:
https://github.com/quipucords/rho/issues/28

We can look at this effort post `master` and `rho-2.0` merge.

### CODE REVIEW

File: library/spit_results.py [hunk 1]
```diff
@@ -18,25 +33,29 @@ def __init__(self, module):
         self.vals = module.params['vals']
 
     def write_to_csv(self):
+        """Method to output report data to file in csv format"""
         f_path = self.file_path
-        f = open(f_path, "w")
+        write_file = open(f_path, "w")
```
Comment (noahl) [inline library/spit_results.py]: I think `with open(f_path, 'w') as write_file` is more Pythonic.

File: rho/cli.py [hunk 2]
```diff
@@ -37,11 +41,17 @@ def __init__(self):
                     self.cli_commands[cmd.name] = cmd
 
     def _add_command(self, cmd):
+        """Method to add a command to the valid commands list
```
Comment (noahl) [inline rho/cli.py]: Consider removing "Method to"

File: test/test_clicommand.py [hunk 3]
```diff
@@ -9,22 +9,25 @@
...
 import unittest
 import sys
+from rho.clicommands import AuthListCommand, ProfileAddCommand, \
+    ProfileListCommand, ScanCommand
 
 
 class HushUpStderr(object):
-
-    def write(self, s):
+    """Class used to quite standard error output"""
```
Comment (noahl) [inline test/test_clicommand.py]: "quite" -> "quiet"

**Review by noahl — COMMENTED** [review]
A few minor points, otherwise looks good to me.

File: library/spit_results.py (hunk 1 above)
Comment (chambridge) [reply library/spit_results.py]: done

File: rho/cli.py (hunk 2 above)
Comment (chambridge) [reply rho/cli.py]: done.

File: test/test_clicommand.py (hunk 3 above)
Comment (chambridge) [reply test/test_clicommand.py]: fixed.

**Pull request merged**